build.sbt

```diff
@@ -5,28 +5,29 @@ description := "Scala client to consume JSON home documents"
 
 organization := "de.kaufhof"
 
-version := "2.2.0"
+version := "2.2.1"
```

Comment:
Hey guys, great to see activity here! And +1 for updating the version separately from features/fixes. 

I hope you're well, Martin